GVFS/GVFS.Platform.Mac/MacFileSystem.cs

```diff
@@ -32,20 +32,21 @@ public override bool IsSocket(string fileName)
 
         public override bool IsFileSystemSupported(string path, out string error)
         {
-            string lowerCaseFile = Guid.NewGuid().ToString().ToLower();
-            string upperCaseFile = lowerCaseFile.ToUpper();
             error = null;
 
             try
             {
-                File.Create(Path.Combine(path, lowerCaseFile));
-                if (File.Exists(Path.Combine(path, upperCaseFile)))
+                string lowerCaseFilePath = Path.Combine(path, Guid.NewGuid().ToString().ToLower());
+                string upperCaseFilePath = lowerCaseFilePath.ToUpper();

```

Comment:
This shouldn't make a difference but it is now upper casing the entire path instead of just the filename.  It should be a valid test for whether the file system is case sensitive or not still though so it should still work.

Reply:
Yeah, I decided the code was simpler to upper-case everything in the path.